### DISCUSSION TITLE
Installation on a Cray computer

### DISCUSSION THREAD

**Discussion by carlos9917** [vlang/v#15546] [Installation Issues]
Hello,
maybe this was answered before, but could not find it.
Has anyone compiled the vlang in a Cray?
Loading gnu compilers I get this message

cd ./vc && git clean -xf && git pull --quiet
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=gnu99 -w -o v1.exe ./vc/v.c -lm -lpthread 
./v1.exe -no-parallel -o v2.exe  cmd/v
GNUmakefile:96: recipe for target 'all' failed
make: *** [all] Illegal instruction

Thanks

Carlos

**Comment by JalonSolov** [comment]
I don't recall anyone mentioning a Cray before.  Not many have access to one.  :-)

**Comment by spytheman** [comment]
Try `export VFLAGS='-cc cc'`, before `make` .

**Reply by carlos9917** [reply]
thanks a lot for the suggestion.Unfortunately that didn't work.

**Reply by JalonSolov** [reply]
Try an explicit compiler name... whichever you have installed.  Like `-cc gcc` or `-cc clang`, etc.
